### PR TITLE
Fixed DateAndTimePicker such that it uses 24h

### PR DESCRIPTION
**Pull request by molleer**
@Portals Please accept and push to npmjs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,17 +38,19 @@
         "lodash": "4.17.21",
         "prop-types": "15.7.2",
         "react-is": "17.0.1",
-        "react-markdown": "4.3.1",
+        "react-markdown": "7.0.1",
```
Comment (Portals) [inline package.json]: Will this break stuff?

Comment (molleer) [reply package.json]: It did break, but updating to `5.0.6` does not break

Comment (Portals) [reply package.json]: Why update it at all?

Comment (molleer) [reply package.json]: `npm install` fails otherwise

Comment (Portals) [reply package.json]: With what error?

Comment (molleer) [reply package.json]:
```
▶ npm install                          
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @cthit/react-digit-components@1.0.3
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   dev react@"17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.0.0 || ^16.0.0" from react-markdown@4.3.1
npm ERR! node_modules/react-markdown
npm ERR!   react-markdown@"4.3.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/molle/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/molle/.npm/_logs/2021-10-02T12_06_54_884Z-debug.log
```

Comment (Portals) [reply package.json]: Ah react 17, weird Eric didn't notice it before.

**Pull request merged**